save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/VulnerabilityProject.kt

```diff
@@ -0,0 +1,39 @@
+package com.saveourtool.save.entities.vulnerabilities
+
+import com.saveourtool.save.entities.vulnerability.VulnerabilityProjectDto
+import com.saveourtool.save.spring.entity.BaseEntityWithDto
+
+import com.fasterxml.jackson.annotation.JsonBackReference
+
+import javax.persistence.*
+
+/**
+ * @property name
+ * @property url
+ * @property isOpen
+ * @property vulnerability
+ **/
```

Comment:
Please update KDoc as it is not obvious why you want 3 different entites

Reply:
Fixed